go.mod

```diff
@@ -383,9 +379,6 @@ require (
 )
 
 replace (
-	// geth wants v2.3.4 but that is incompatible with github.com/cometbft/cometbft v0.37.5 which when bumped is incompatible with github.com/cosmos/cosmos-sdk
-	// This line can be removed after these imports are bumped or removed.
-	github.com/btcsuite/btcd/btcec/v2 => github.com/btcsuite/btcd/btcec/v2 v2.3.2
 	// replicating the replace directive on cosmos SDK
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

Comment:
We are stuck with this for some reason. Cosmos and geth both use `github.com/gogo/protobuf v1.3.2`, without any `replace`, but something else is trying to force us to `v1.3.3` still... :thinking: